examples/llama-bench/llama-bench.cpp

```diff
@@ -916,6 +918,9 @@ static void llama_null_log_callback(enum llama_log_level level, const char * tex
 }
 
 int main(int argc, char ** argv) {
+    // try to set locale for unicode characters in markdown
+    setlocale(LC_CTYPE, ".UTF-8");
```

Comment:
On non-Windows platforms this is going to fail and return NULL. I think we should wrap this with `#ifdef _WIN32`.

Reply:
Yes, but if it fails nothing changes anyway, so I don't see the point.

Reply:
> Yes, but if it fails nothing changes anyway, so I don't see the point.

I can confirm, nothing happens on Linux, locale stays what it was before the `setlocale(LC_CTYPE, ".UTF-8");` call, no exceptions, no errors.